sysl2/sysl/cmd_runner.go

```diff
@@ -84,6 +84,13 @@ func EnsureFlagsNonEmpty(cmd *kingpin.CmdClause, excludes ...string) {
 	}
 	fn := func(c *kingpin.ParseContext) error {
 		var errorMsg strings.Builder
+		for _, elem := range c.Elements {
+			if f, _ := elem.Clause.(*kingpin.FlagClause); f != nil {
+				if f.Model().Name == "help" {
```

Comment:
nitpicky but what about
```go
if f, _ := elem.Clause(*kingpin.FlagClause); f != nil && f.Model.Name == "help"{
    return nil
}
```

Reply:
or is that too long?

Reply:
indeed, very nitpicky, but maybe slightly moore readable.